### PR TITLE
Adding protocol

### PR DESCRIPTION
**Pull request by Ciejo**

### CODE REVIEW

**Review by juanmayoraljob — APPROVED** [review]

File: variables.tf [hunk 1]
```diff
@@ -60,4 +60,10 @@ variable "create_alias_records" {
   type        = bool
   description = "Put in true or false if you want to create alias records"
   default     = true
+}
+
+variable "minimum_protocol_version" {
+  type = string
+  description = "Protocol version"
+  default = "TLSv1"
```
Comment (qemanuel) [inline variables.tf:68]: @Ciejo Please, fix identation

File: main.tf [hunk 2]
```diff
@@ -83,6 +83,7 @@ resource "aws_cloudfront_distribution" "default" {
   viewer_certificate {
     acm_certificate_arn = var.certificate_arn
     ssl_support_method  = "sni-only"
+    minimum_protocol_version = var.minimum_protocol_version
```
Comment (qemanuel) [inline main.tf:86]: @Ciejo Please, fix identation

File: README.md [hunk 3]
```diff
@@ -30,6 +30,7 @@ module "s3_website" {
   custom_subdomain     = "awesome"
   aliases              = ["tooawesome.nimbux911.com", "reallyawesome.nimbux911.com"]
   create_alias_records = true
+  minimum_protocol_version = "TLSv1"
```
Comment (qemanuel) [inline README.md:33]: @Ciejo Please, fix identation

**Review by qemanuel — APPROVED** [review]
@Ciejo Everything looks good but identantion. Please fix that before merging it

**Review by qemanuel — APPROVED** [review]
@Ciejo Everything looks good but identantion. Please fix that before merging it

**Review by qemanuel — APPROVED** [review]

**Review by qemanuel — APPROVED** [review]

**Review by qemanuel — APPROVED** [review]
LGTM

**Review by qemanuel — APPROVED** [review]

**Review by qemanuel — APPROVED** [review]
Approved

**Review by qedgardo — APPROVED** [review]

**Pull request merged**